docs/core/runtime-config/garbage-collector.md

```diff
@@ -313,18 +313,19 @@ For more information about some of these settings, see the [Middle ground betwee
 
 ### Affinitize ranges
 
-- Specifies the list of processors to use for garbage collector threads.
+- Specifies the list of processors to use for garbage collector threads. This can either 
```

Comment:
It looks like the added sentence is not complete

Reply:
oops, I meant to delete that change.